### ISSUE TITLE
more services and packages for a minimal yet suite workstation

### ISSUE THREAD

**Issue by mh-firouzjah** [D3SOX/arch-guide#138]
Hi there 👋🏻

## TLDR; 

please have a review of `pkglist.txt` and `services.txt` files from [my repo](https://github.com/mh-firouzjah/archlinux-installation-guide) 🤝🏻.

## Long story

I've been searching for a complete arch linux post-installation guide and I've found [your awesome website](https://arch.d3sox.me/installation) which was very useful for me. also I was working on gathering some useful/required packages and services to bring a fresh arch linux install to reliable and stable workstation for myself. I've been a distro hopper for a long time and I'm sick and tired of testing each and every distro out there, so I come to arch and want to make a minimal yet powerful install for my need but I love to see if this could help any other Arch lover as well.

**Comment by D3SOX** [comment]
Hi, I've had a cursory look over your list but I didn't find any specific packages that are not mentioned in my guide. Could you please elaborate a bit more which packages for example should be added and what you think where to integrate them in the guide.
